### DISCUSSION TITLE
Error (invalid escape character \u) while trying to create a message in vrl console

### DISCUSSION THREAD

**Discussion by mans2singh** [vectordotdev/vector#14700] [Q&A] [answered]
Hey Folks:

I am trying to go through the examples in vector docs and saw an example [with embedded unicode char](https://vector.dev/docs/reference/vrl/examples/#multiple-parsing-strategies).  When I try to enter it in vrl console, it gives me an error as shown below.  If I create a message with simple text it succeeds.  

Can you please let me know what could be the issue ?  

Thanks

> $ message_test = {
  "message": "\u003c102\u003e1 2020-12-22T15:22:31.111Z vector-user.biz su 2666 ID389 - Something went wrong"
}

error[E202]: syntax error
  ┌─ :1:1
  │
1 │ ╭ message_test = {
2 │ │   "message": "\u003c102\u003e1 2020-12-22T15:22:31.111Z vector-user.biz su 2666 ID389 - Something went wrong"
3 │ │ }
  │ ╰─^ unexpected error: invalid escape character: \u
  │
  = see language documentation at https://vrl.dev
  = try your code in the VRL REPL, learn more at https://vrl.dev/examples

**Comment by jszwedko** [comment, marked as answer]
Hi @mans2singh ! Unfortunately unicode escapes aren't supported in VRL string literals yet. This is being tracked by https://github.com/vectordotdev/vector/issues/11926

**Reply by mans2singh** [reply]
Thanks @jszwedko for your help.
